vnext/Microsoft.ReactNative/Fabric/Composition/Composition.Input.h

```diff
@@ -28,4 +33,123 @@ struct KeyRoutedEventArgs : winrt::implements<
   winrt::Windows::UI::Core::CorePhysicalKeyStatus m_keyStatus;
 };
 
+struct PointerPointProperties : PointerPointPropertiesT<PointerPointProperties> {
+  PointerPointProperties(const winrt::Windows::UI::Input::PointerPointProperties &ppp);
+
+  PointerPointProperties(
+      bool isBarrelButtonPressed,
+      bool isCanceled,
+      bool isEraser,
+      bool isHorizontalMouseWheel,
+      bool isInRange,
+      bool isInverted,
+      bool isLeftButtonPressed,
+      bool isMiddleButtonPressed,
+      bool isPrimary,
+      bool isRightButtonPressed,
+      bool isXButton1Pressed,
+      bool isXButton2Pressed,
+      bool touchConfidence,
+      int32_t mouseWheelDelta,
+      float orientation,
+      PointerUpdateKind pointerUpdateKind,
+      float pressure,
+      float twist,
+      float xTilt,
+      float yTilt);
+
+  winrt::Windows::Foundation::Rect ContactRect() noexcept;
+  bool IsBarrelButtonPressed() noexcept;
+  bool IsCanceled() noexcept;
+  bool IsEraser() noexcept;
+  bool IsHorizontalMouseWheel() noexcept;
+  bool IsInRange() noexcept;
+  bool IsInverted() noexcept;
+  bool IsLeftButtonPressed() noexcept;
+  bool IsMiddleButtonPressed() noexcept;
+  bool IsPrimary() noexcept;
+  bool IsRightButtonPressed() noexcept;
+  bool IsXButton1Pressed() noexcept;
+  bool IsXButton2Pressed() noexcept;
+  int32_t MouseWheelDelta() noexcept;
+  float Orientation() noexcept;
+  PointerUpdateKind PointerUpdateKind() noexcept;
+  float Pressure() noexcept;
+  bool TouchConfidence() noexcept;
+  float Twist() noexcept;
+  float XTilt() noexcept;
+  float YTilt() noexcept;
+
+ private:
+  winrt::Windows::UI::Input::PointerPointProperties m_systemPPP{nullptr};
```

Comment:
nit: maybe rename to something less abbreviated?

```suggestion
  winrt::Windows::UI::Input::PointerPointProperties m_sysPointerPointProps{nullptr};
```